src/app/static/src/app/components/header/FileMenu.vue

```diff
@@ -162,6 +166,16 @@
                     }
                 }
             },
+            clearLoadZipInput() {
+                // clearing value because browser does not
+                // allow selection of the same file twice
+                const input = this.$refs.loadZip as HTMLInputElement
+                input.value = ""
+            },
```

Comment:
I have added this fix to the bug discovered by @EmmaLRussell which originated from the existing code. Browsers have implemented guard against uploading the same input file more than once. This could have been resolved easily with `onclick() => setValue to null` but we do not directly interact with this input element as we trigger the input via fileMenu link.